Source/Core/MapProjection.js

```diff
@@ -31,6 +43,20 @@ define([
          */
         ellipsoid : {
             get : DeveloperError.throwInstantiationError
+        },
+        /**
+         * Gets whether or not the projection evenly maps meridians to vertical lines.
+         * Projections that evenly map meridians to vertical lines (such as Web Mercator and Geographic) do not need
+         * addition 2D vertex attributes and are more efficient to render.
+         *
+         * @memberof MapProjection.prototype
+         *
+         * @type {Boolean}
+         * @readonly
+         * @private
+         */
+        isNormalCylindrical : {
```

Comment:
Why is this private?  It's part of the public `MapProjection` interface.  There shouldn't be a need for deprecation here, as implementing your own interface was never officially supported so this isn't a breaking change, but making this `private` is the wrong thing to do.